pkg/downloader/uri.go

```diff
@@ -225,7 +225,7 @@ func (s URI) ResolveURL() string {
 		repo := repoPieces[1]
 
 		branch := "main"
-		filepath := repoPieces[2]
+		filepath := strings.Join(repoPieces[2:], "/")
```

Comment:
would be great if we could cover this in our unit tests in `pkg/downloader/uri_test.go`